spec/rspec/rails/matchers/active_job_spec.rb

```diff
@@ -207,5 +207,13 @@ def to_global_id(options = {})
         }.to have_enqueued_job(hello_job).with(42).on_queue("low").at(date).exactly(2).times
       }.to raise_error(message)
     end
+
+    it "throws descriptive error when no test adapter set" do
+      ActiveJob::Base.queue_adapter = :inline
```

Comment:
And as we run tests randomly you can't guarantee this, we should reset it back to it's original setting (not :test specifically).